### ISSUE TITLE
Issue with pydantic

### ISSUE THREAD

**Issue by RomLecat** [devopsarr/sonarr-py#52]
Hello,

Per the requirements, pydantic is v1.10.13 on Python 3.11.

```
$ pip freeze
aenum==3.1.15
certifi==2023.11.17
charset-normalizer==3.3.2
idna==3.6
netifaces==0.11.0
packaging==23.2
pydantic==1.10.13
python-dateutil==2.8.2
PyYAML==6.0.1
pyzabbix==1.3.1
radarr-py @ git+https://github.com/devopsarr/radarr-py@06685bf68fbc065bc4563ca33eb36db0e672b46d
requests==2.31.0
six==1.16.0
sonarr-py @ git+https://github.com/devopsarr/sonarr-py@56153ead9ab42b6924fcae63fd26444b85c87867
typing_extensions==4.9.0
urllib3==2.1.0
```

Which causes an issue:

```
Traceback (most recent call last):
  File "/opt/zabbix-tools/arr_agent.py", line 9, in <module>
    import sonarr
  File "/opt/venv/lib/python3.11/site-packages/sonarr/__init__.py", line 22, in <module>
    from sonarr.api.api_info_api import ApiInfoApi
  File "/opt/venv/lib/python3.11/site-packages/sonarr/api/__init__.py", line 46, in <module>
    from sonarr.api.log_file_api import LogFileApi
  File "/opt/venv/lib/python3.11/site-packages/sonarr/api/log_file_api.py", line 22, in <module>
    from pydantic import field_validator
ImportError: cannot import name 'field_validator' from 'pydantic' (/opt/venv/lib/python3.11/site-packages/pydantic/__init__.cpython-311-x86_64-linux-gnu.so)
```

Thanks

**Comment by Fuochi** [comment]
Hi @RomLecat,
Thanks for opening the issue. I'm planning a migration to pydantic 2.x in February. I'll make sure next release won't have this problem.

**Comment by Fuochi** [comment]
Hi @RomLecat this should be now fixed in v1.0.1

**Comment by RomLecat** [comment]
Hi,

It works fine now, thanks !
